utils/config_generator.py

```diff
@@ -45,6 +45,13 @@
     "lithuanian_legislation_seimas_lithuania",
     # Fails to load from OPUS.
     "SPC",
+    # MTdata duplicates Flores that we pull directly
+    "flores101_dev",
+    "flores101_devtest",
+    "flores200_dev",
+    "flores200_devtest",
+    # Skip OPUS WMT news test sets
```

Comment:
Suggestion: This comment says the "what", but not the "why". Perhaps it could be rephrased to something like: "OPUS WMT is used in our evaluations and shouldn't be used for training".